src/HomogeneousCalorimeter_geo.cpp

```diff
@@ -288,38 +303,67 @@ static std::tuple<int, std::pair<int, int>> add_12surface_disk(Detector& desc, A
 
 
 
-
   //=====================================================================
   // Placing The Modules
-  // Since the inner and outer porfile is not the circle shape,
-  // which means the modules placements can't be simply done by
-  // fillRectangles functions. I hardcode the additional positions
-  // at backward_PbWO4.xml to fill all the gap between circular
-  // placement and supporting structures.
   //=====================================================================
 
-  // retrieve the max and min position of modules
-  auto [minl_ptsx, maxl_ptsx] = std::minmax_element(points.begin(), points.end(), [](auto a, auto b){ return a.x() < b.x(); });
-  auto [minl_ptsy, maxl_ptsy] = std::minmax_element(points.begin(), points.end(), [](auto a, auto b){ return a.y() < b.y(); });
+  auto test_points = epic::geo::fillRectangles({half_modx, half_mody}, modSize.x(), modSize.y(), 0., (rmax/std::cos(Prot)), phimin, phimax);
+
+  std::pair<double, double> c1 (0., 0.);
+  auto polyVertex = epic::geo::getPolygonVertices(c1, (rmax/std::cos(Prot)), M_PI/12., 12);
+  std::vector<epic::geo::Point> Out_vertices, In_vertices;
```

Comment:
Local variables lowercase.
```suggestion
  std::vector<epic::geo::Point> out_vertices, in_vertices;
```

Reply:
changed in the last commit